src/sql/base/browser/ui/table/plugins/checkboxSelectColumn.plugin.ts

```diff
@@ -14,6 +15,20 @@ export interface ICheckboxSelectColumnOptions extends Slick.PluginOptions, IChec
 	toolTip?: string;
 	width?: number;
 	title?: string;
+	columnIndex?: number;
+	actionOnCheck?: ActionOnCheck;
+}
+
+// Actions expected on checkbox click
+export enum ActionOnCheck {
+	default = 0,
```

Comment:
selectRow

Reply:
yes sure, will change. Did that only to make it sound generic.